### PR TITLE
Docs: Add configuration comments in examples

### PR DESCRIPTION
**Pull request by yeonjuan**
#### Prerequisites checklist

- [x] I have read the [contributing guidelines](https://github.com/eslint/eslint/blob/master/CONTRIBUTING.md).

#### What is the purpose of this pull request? (put an "X" next to an item)

[x] Documentation update
[ ] Bug fix ([template](https://raw.githubusercontent.com/eslint/eslint/master/templates/bug-report.md))
[ ] New rule ([template](https://raw.githubusercontent.com/eslint/eslint/master/templates/rule-proposal.md))
[ ] Changes an existing rule ([template](https://raw.githubusercontent.com/eslint/eslint/master/templates/rule-change-proposal.md))
[ ] Add autofixing to a rule
[ ] Add a CLI option
[ ] Add something to the core
[ ] Other, please explain:

#### What changes did you make? (Give an overview)

I added missing configuration comments in the documents. It's quite useful for a quick demo :)

#### Is there anything you'd like reviewers to focus on?

no.

### CODE REVIEW

**Review by anikethsaha — APPROVED** [review]

File: docs/rules/no-unmodified-loop-condition.md [hunk 1]
```diff
@@ -29,6 +29,8 @@ If a reference is inside of a dynamic expression (e.g. `CallExpression`,
 Examples of **incorrect** code for this rule:
 
 ```js
+/*eslint no-unmodified-loop-condition: "error"*/
+
 while (node) {
     doSomething(node);
 }
```
Comment (mdjermanovic) [inline docs/rules/no-unmodified-loop-condition.md:36]: This doesn't seem to work in the demo (at least when I try), maybe the variables have to be declared?

@yeonjuan if this really should be fixed, what do you think about doing that in this PR or in another PR?

**Review by mdjermanovic — COMMENTED** [review]
Nice catch!

The PR is correct, but it looks like some examples were not :)

File: docs/rules/no-unmodified-loop-condition.md (hunk 1 above)
Comment (yeonjuan) [reply docs/rules/no-unmodified-loop-condition.md]: > if this really should be fixed, what do you think about doing that in this PR or in another PR?
@mdjermanovic   I think doing in this PR nice. I fixed it. Thanks 👍

Comment (mdjermanovic) [reply docs/rules/no-unmodified-loop-condition.md]: Thanks! The example with for-loop also doesn't work, but that seems to be a more complex problem for another PR. I'm not sure what was the expected behavior.

**Review by mdjermanovic — APPROVED** [review]
LGTM, thanks!

**Pull request merged**